app/gui/view/src/project/project_view_top_bar.rs

```diff
@@ -0,0 +1,69 @@
+//! Defines a UI container for the window control buttons and the "go to dashboard" button. This is
+//! merely here to make use of the auto-layout functionality.
+
+use ensogl::prelude::*;
+
+use enso_config::ARGS;
+use ensogl::application::Application;
+use ensogl::display;
+
+
+
+mod go_to_dashboard_button;
+pub mod window_control_buttons;
+
+
+
+// =================
+// === Constants ===
+// =================
+
+/// The gap in pixels between the various components of the project view top bar.
+const GAP: f32 = 18.0;
+/// The padding around the project view top bar.
+const PADDING: f32 = 18.0;
```

Comment:
I don't understand this - what is this padding? What does it mean that this is "around" the project view top bar? Also, what is "project view top bar" and which gap in Figma this 18 pixels space refers to?

Reply:
I've changed it to be a bit more accurate now. Have another look. `PADDING_LEFT` is now the padding to the left of the top bar (and changed to 19px as in the Figma design). "Project view top bar" is the name you decided on during our long call two weeks ago.